src/raw_node.rs

```diff
@@ -130,6 +132,18 @@ impl Ready {
             entries: raft.raft_log.unstable_entries().unwrap_or(&[]).to_vec(),
             ..Default::default()
         };
+        if let Some(e) = rd.entries.first() {
+            for i in (0..raft.conf_states().len()).rev() {
```

Comment:
Why not use `Iterator::find`? It seems more understandable.

Reply:
Seems it's better to find it from tail to head because there could be many unapplied entries in `conf_states`.

Reply:
You mean `entries.iter().rev().position()`?